package.json

```diff
@@ -2,7 +2,8 @@
   "name": "katex",
   "version": "0.11.0-pre",
   "description": "Fast math typesetting for the web.",
-  "main": "dist/katex.js",
+  "main": "dist/katex",
+  "module": "dist/katex.mjs",
```

Comment:
Does node use this when doing `import ... from "katex"` when the `--harmony` flag is enabled?

Reply:
@kevinbarabash Node only uses `main` field, which is extensionless, and when modules are enabled(currently `--experimental-modules`), Node resolves it to `dist/katex.mjs`.